doc/user/content/sql/system-catalog/mz_internal.md

```diff
@@ -402,6 +402,20 @@ inputs.
 | `replica_id` | [`text`]    | The ID of a cluster replica. |
 | `hydrated`   | [`boolean`] | Whether the object is hydrated on the replica. |
 
+## `mz_license_keys`
+
+The `mz_license_keys` table describes the license keys which are currently in
+use.
+
+<!-- RELATION_SPEC mz_internal.mz_license_keys -->
+| Field            | Type                         | Meaning  |
+| -----------      | -----------                  | -------- |
+| `id`             | [`text`]                     | The identifier of this particular license key. |
+| `organization`   | [`text`]                     | The name of the organization that this license key was issued to. |
```

Comment:
Just for my edification ... does this organization map to something like the company name associated with the account setting?

Reply:
yeah, it's just an arbitrary string identifying the organization that this license key was issued to. for now, it's just the frontegg tenant id for the customer, but it could be anything (i think for community edition license keys, it will probably be the user's email address, for instance)